<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkToFConnectViewControls</class>
 <widget class="QWidget" name="QmitkToFConnectViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>452</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QmitkServiceListWidget" name="m_DeviceFactoryServiceListWidget" native="true"/>
   </item>
   <item>
    <widget class="QmitkServiceListWidget" name="m_ConnectedDeviceServiceListWidget" native="true"/>
   </item>
   <item>
    <widget class="QPushButton" name="m_GetRegistered">
     <property name="text">
      <string>Get Registered Factories</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_GetConnect">
     <property name="text">
      <string>Get Connected Devices</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_ConnectDevice">
     <property name="text">
      <string>Connect Device</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkToFRecorderWidget" name="m_TofRecorderWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkServiceListWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkServiceListWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkToFRecorderWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkToFRecorderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
